src/routes/(auth)/phrases/[phraseId]/+server.ts

```diff
@@ -0,0 +1,58 @@
+import { deletePhrase } from "$lib/phrases/services/delete-phrase";
+import { getPhrase } from "$lib/phrases/services/get-phrase";
+import { updatePhrase } from "$lib/phrases/services/update-phrase";
+import { getSupabase } from "@supabase/auth-helpers-sveltekit";
+import { error, type RequestHandler } from "@sveltejs/kit";
+
+export const GET: RequestHandler = async (event) => {
+  const { params } = event;
+  if (params.phraseId == null) {
+    throw error(404);
+  }
+  const phraseId = BigInt(params.phraseId);
```

Comment:
**nitpick (blocking):** Conversion trigger an exception.

[According to the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) this conversion may trigger an exception when provided a non numeric char, making the next check not necessary. Same for all of this conversions.